server/repository/src/migrations/v1_01_01/mod.rs

```diff
@@ -0,0 +1,116 @@
+            CREATE INDEX "index_invoice_line_stock_line_id_fkey" ON "invoice_line" ("stock_line_id");
+            CREATE INDEX "index_invoice_line_item_id_fkey" ON "invoice_line" ("item_id");
+            CREATE INDEX "index_invoice_line_invoice_id_fkey" ON "invoice_line" ("invoice_id");
+            CREATE INDEX "index_activity_log_store_id_fkey" ON "activity_log" ("store_id");
+            CREATE INDEX "index_stocktake_inventory_adjustment_id_fkey" ON "stocktake" ("inventory_adjustment_id");
+            CREATE INDEX "index_stocktake_store_id_fkey" ON "stocktake" ("store_id");
+            CREATE INDEX "index_location_store_id_fkey" ON "location" ("store_id");
+            CREATE INDEX "index_user_permission_store_id_fkey" ON "user_permission" ("store_id");
+            CREATE INDEX "index_user_permission_user_id_fkey" ON "user_permission" ("user_id");
+            CREATE INDEX "index_store_name_id_fkey" ON "store" ("name_id");
+            CREATE INDEX "index_user_store_join_store_id_fkey" ON "user_store_join" ("store_id");
+            CREATE INDEX "index_user_store_join_user_id_fkey" ON "user_store_join" ("user_id");
+            CREATE INDEX "index_invoice_store_id_fkey" ON "invoice" ("store_id");
+            CREATE INDEX "index_invoice_name_store_id_fkey" ON "invoice" ("name_store_id");
+            CREATE INDEX "index_invoice_name_id_fkey" ON "invoice" ("name_id");
+            CREATE INDEX "index_location_movement_stock_line_id_fkey" ON "location_movement" ("stock_line_id");
+            CREATE INDEX "index_location_movement_location_id_fkey" ON "location_movement" ("location_id");
+            CREATE INDEX "index_location_movement_store_id_fkey" ON "location_movement" ("store_id");
+            CREATE INDEX "index_master_list_name_join_master_list_id_fkey" ON "master_list_name_join" ("master_list_id");
+            CREATE INDEX "index_master_list_name_join_name_id_fkey" ON "master_list_name_join" ("name_id");
+            CREATE INDEX "index_item_unit_id_fkey" ON "item" ("unit_id");
+            CREATE INDEX "index_name_store_join_store_id_fkey" ON "name_store_join" ("store_id");
+            CREATE INDEX "index_name_store_join_name_id_fkey" ON "name_store_join" ("name_id");
+            CREATE INDEX "index_stock_line_location_id_fkey" ON "stock_line" ("location_id");
+            CREATE INDEX "index_stock_line_store_id_fkey" ON "stock_line" ("store_id");
+            CREATE INDEX "index_stock_line_item_id_fkey" ON "stock_line" ("item_id");
+            CREATE INDEX "index_master_list_line_master_list_id_fkey" ON "master_list_line" ("master_list_id");
+            CREATE INDEX "index_master_list_line_item_id_fkey" ON "master_list_line" ("item_id");
+            CREATE INDEX "index_requisition_line_requisition_id_fkey" ON "requisition_line" ("requisition_id");
+            CREATE INDEX "index_requisition_name_id_fkey" ON "requisition" ("name_id");
```

Comment:
perhaps should move down, under `-- Requisition`

Reply:
Could do, the first section combines all of the tables where index does not exist for references (relation), it was auto generated with a script in the comment, so they are just grouped by relation/reference.

Just a general note, with migrations, we wouldn't really be able to make sense of what the schema is by looking at migrations themselves. Could only really see the real state of DB through some tool (pgAdmin etc..), so I was thinking that migration files won't really be used to make sense of schema itself thus don't really need to be exceptionally tidy. There is a carry over in one of the issue for schema spy (for quick and easy way to see state of schema, including indexes)